website/content/docs/troubleshoot/faq.mdx

```diff
@@ -47,10 +47,15 @@ For more information on dynamic host catalogs, please see:
 - [Dynamic Host Catalogs on Azure](/boundary/tutorials/access-management/azure-host-catalogs)
 
 #### Q: Session management capabilities - Does Boundary support live session monitoring and termination?
-**Session Logging/Monitoring:** Supported. Boundary creates a session log of all sessions created between identities and targets that have been onboarded to 
-Boundary. You can learn how to monitor these sessions in this [tutorial](/boundary/tutorials/getting-started/getting-started-connect#manage-sessions). 
-Boundary supports audit logs for [Boundary OSS](/boundary/tutorials/oss-configuration/event-logging) and [audit log streaming](/hcp/docs/boundary/audit-logging) 
+**Session Logging/Monitoring:** Supported. Boundary creates a session log of all sessions created between identities and targets that have been onboarded to
+Boundary. You can learn how to monitor these sessions in this [tutorial](/boundary/tutorials/getting-started/getting-started-connect#manage-sessions).
+Boundary supports audit logs for [Boundary OSS](/boundary/tutorials/oss-configuration/event-logging) and [audit log streaming](/hcp/docs/boundary/audit-logging)
 for HCP Boundary. Audit logs for both distributions can be exported to SIEM or BI tools.
 
-**Session Termination:** Supported. Session termination for Boundary administrators is a supported capability, as demonstrated in this 
+**Session Termination:** Supported. Session termination for Boundary administrators is a supported capability, as demonstrated in this
 [tutorial](/boundary/tutorials/getting-started/getting-started-connect#manage-sessions).
+
+#### Q: As an AWS user, can I use IAM role to configure Boundary's dynamic host catalog?
+At this time, it is not possible to configure Boundary's dynamic host catalog using IAM role.
+You can configure the dynamic host catalog using IAM user, however.
+For more information, refer to the [Dynamic host catalogs on AWS](/boundary/tutorials/access-management/aws-host-catalogs) tutorial.
```

Comment:
hmm, wondering if IAM roles/users needs to be plural or singular? I hear it most often as "IAM roles" or "an IAM role"

```suggestion
At this time, it is not possible to configure Boundary's dynamic host catalog using IAM roles.
You can configure the dynamic host catalog using an IAM user, however.
For more information, refer to the [Dynamic host catalogs on AWS](/boundary/tutorials/access-management/aws-host-catalogs) tutorial.
```